Projects/hCore/Sources/SafariView+SwiftUI.swift

```diff
@@ -0,0 +1,65 @@
+import Flow
+import Form
+import Presentation
+import SafariServices
+import SwiftUI
+import UIKit
+import WebKit
+
+public struct SafariView: UIViewControllerRepresentable {
+    public typealias UIViewControllerType = ReloadingSafariViewController
+
+    @Binding var url: URL?
+
+    public init(
```

Comment:
is there no native SwiftUI way to do safari related stuff yet? Just curious

Reply:
Sadly not 🥲